addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -1123,6 +1124,64 @@ var SummernoteManager = Class.extend(mixins.EventDispatcherMixin, {
         core.bus.off('media_dialog_demand', this, this._onMediaDialogDemand);
     },
 
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Create/Update cropped attachments.
+     *
+     * @private
+     * @param {jQuery} $editable
+     * @returns {Deferred}
+     */
+    _saveCroppedImages: function ($editable) {
+        var defs = _.map($editable.find('.o_cropped_img_to_save'), function (croppedImg) {
+            var $croppedImg = $(croppedImg);
+            $croppedImg.removeClass('o_cropped_img_to_save');
+
+            var resModel = $croppedImg.data('crop:resModel');
+            var resID = $croppedImg.data('crop:resID');
+            var cropID = $croppedImg.data('crop:id');
+            var mimetype = $croppedImg.data('crop:mimetype');
+            var originalSrc = $croppedImg.data('crop:originalSrc');
+
+            var datas = $croppedImg.attr('src').split(',')[1];
+
+            if (!cropID) {
+                var name = originalSrc + '.crop';
+                return rpc.query({
```

Comment:
Can you make the SummernoteManager inherit from the service mixin so that `_rpc` is available instead of using `rpc.query`?

Reply:
Yes Good idea :smiley: